Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Base on offical Node.js Alpine image
+FROM node:current-alpine AS base
+
+# Set working directory
+WORKDIR /base
+
+# Copy package.json and package-lock.json before other files
+# Utilise Docker cache to save re-installing dependencies if unchanged
+COPY package*.json yarn.lock ./ 
+
+# Install dependencies
+RUN yarn install --frozen-lockfile
+
+# Copy all files
+COPY . .
+
+# Build app
+FROM base AS builder
+ENV NODE_ENV=production
+WORKDIR /build
+COPY . .
+COPY --from=base /base/node_modules ./node_modules
+RUN yarn build
+
+# Production image, copy all the files and run next
```

Comment:
```suggestion
# Production image, [DESCRIPTION]
```

Reply:
I'd make it's synchronized to what I've mentioned in earlier comments. What do you think? 🤔 
```suggestion
# Production stage
```

Reply:
SGTM